ui/src/app/pages/analysis/notebook-redirect.spec.tsx

```diff
@@ -28,8 +30,17 @@ describe('NotebookRedirect', () => {
 
   let runtimeStub: RuntimeApiStub;
 
-  const mountedComponent = () => {
-    return mount(<NotebookRedirect/>);
+  const handleUseEffect = async (component) => {
```

Comment:
This is in runtime-utils and runtime-panels too - could it be pulled out into a test utils file?

Reply:
done